library/test/tokens.test.ts

```diff
@@ -952,4 +953,124 @@ describe('Tokens', () => {
       });
     });
   });
+
+  describe('search tokens', () => {
+    it('should search tokens', async () => {
+      const totalItems = chance.integer();
+      const pageNumber = chance.integer();
+      const pageSize = chance.integer();
+      const totalPages = chance.integer();
+
+      const searchRequest = {
+        query: chance.string(),
+        page: chance.integer(),
+        size: chance.integer(),
+      } as SearchTokensRequest;
+
+      /* eslint-disable camelcase */
+      client.onPost('search').reply(
+        200,
+        JSON.stringify({
+          pagination: {
+            total_items: totalItems,
+            page_number: pageNumber,
+            page_size: pageSize,
+            total_pages: totalPages,
+          },
+          data: [],
+        })
+      );
+      /* eslint-enable camelcase */
+
+      expect(await bt.tokens.search(searchRequest)).toStrictEqual({
+        pagination: {
+          totalItems,
+          pageNumber,
+          pageSize,
+          totalPages,
+        },
+        data: [],
+      } as PaginatedList<Token>);
+
+      expect(client.history.post.length).toBe(1);
+      expect(client.history.post[0].url).toStrictEqual('/search');
```

Comment:
shouldn't this url be /tokens/search?

Reply:
it is constructed depending on the client type